yarn.lock

```diff
@@ -35841,4 +35841,4 @@ zone.js@~0.11.4:
   resolved "https://registry.yarnpkg.com/zone.js/-/zone.js-0.11.7.tgz#262194267c7b964e8da77ce16b9fba9bea23cfdc"
   integrity sha512-e39K2EdK5JfA3FDuUTVRvPlYV4aBfnOOcGuILhQAT7nzeV12uSrLBzImUM9CDVoncDSX4brR/gwqu0heQ3BQ0g==
   dependencies:
-    tslib "^2.3.0"
+    tslib "^2.3.0"
```

Comment:
Weird this removed a blank newline at the end of the file.

Reply:
I was the culprit here -- I updated a dev dependency and then had trouble backpedaling. Ended up just copying `yarn.lock` from `develop` over here 🤷‍♀️